tests/test_config.py

```
@@ -0,0 +1,345 @@
+        """Test crawling-specific configuration."""
+        config = CrawlerrSettings(
+            crawl_headless=False,
+            crawl_browser="firefox",
+            crawl_max_pages=2000,
+            crawl_max_depth=5,
+            max_concurrent_crawls=50,
+            crawler_delay=1.0,
+            crawler_timeout=60.0,
+        )
+
+        assert config.crawl_headless == False
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not config.crawl_headless:` for false checks

Replace with `not config.crawl_headless`

(E712)